tiled/server/router.py

```diff
@@ -1374,6 +1356,57 @@ async def put_awkward_full(
     return json_or_msgpack(request, None)
 
 
+@router.patch("/metadata/{path:path}", response_model=schemas.PatchMetadataResponse)
+async def patch_metadata(
+    request: Request,
+    body: schemas.PatchMetadataRequest,
+    validation_registry=Depends(get_validation_registry),
+    settings: BaseSettings = Depends(get_settings),
+    entry=SecureEntry(scopes=["write:metadata"]),
+):
+    if not hasattr(entry, "replace_metadata"):
+        raise HTTPException(
+            status_code=HTTP_405_METHOD_NOT_ALLOWED,
+            detail="This node does not support update of metadata.",
+        )
+
+    if request.headers["content-type"] == "application/json-patch+json":
+        metadata = apply_json_patch(
+            entry.metadata(), body.patch if body.patch is not None else []
```

Comment:
This is a nice pattern when an empty collection (`[]` in this case) or `None` should both lead to the same result.

```suggestion
            entry.metadata(), (body.patch or [])
```